src/legacy/components/analytics/events/transactionEvents.ts

```diff
@@ -58,7 +58,7 @@ export function approvalAnalytics(action: ApprovalAction, label?: string, value?
   })
 }
 
-export type SwapAction = 'Send' | 'Error' | 'Reject' | 'Bundle Approve and Swap'
+export type SwapAction = 'Send' | 'Error' | 'Reject' | 'Bundle Approve and Swap' | 'Bundle Wrap Approve and Swap'
```

Comment:
leave it as it is if you think its better, alternativelly, feels simpler `Bundled Eth Flow` (easier to differenciate with the other)